### PR TITLE
feat(bottlecap): add `runtime` tag

### PR DESCRIPTION
**Pull request by alexgallotta**

### CODE REVIEW

File: bottlecap/src/tags/lambda/tags.rs [hunk 1]
```diff
@@ -113,6 +116,16 @@ fn tags_from_env(
     if let Ok(memory_size) = std::env::var(MEMORY_SIZE_VAR) {
         tags_map.insert(MEMORY_SIZE_KEY.to_string(), memory_size);
     }
+    if let Ok(exec_runtime_env) = std::env::var(RUNTIME_VAR) {
+        // AWS_Lambda_java8
```
Comment (astuyve) [inline bottlecap/src/tags/lambda/tags.rs:120]: if this takes ~25ms can we add an opt-out for people who may know what their runtime is and not care?

Comment (alexgallotta) [reply bottlecap/src/tags/lambda/tags.rs:120]: it seems the 25ms delay is coded in goagent (more details in the jira https://datadoghq.atlassian.net/browse/SVLS-4904 ) so we are not adding anything here

**Comment by alexgallotta** [comment]
It seems serialized-size byte tests are failing both here and in main.
I am not sure if it depends on my arch building protobuf or some other changes. If anyone else can replicate the green test or the failing it would be extremely helpful

File: bottlecap/src/tags/lambda/tags.rs [hunk 2]
```diff
@@ -1,13 +1,16 @@
+use crate::config;
+use log::warn;
```
Comment (duncanista) [inline bottlecap/src/tags/lambda/tags.rs:2]: don't use `log` crate

File: bottlecap/src/tags/lambda/tags.rs [hunk 3]
```diff
@@ -132,6 +145,41 @@ fn tags_from_env(
     tags_map
 }
 
+fn resolve_provided_runtime(path: &str) -> String {
+    let path = Path::new(path);
+
+    let file = match File::open(path) {
+        Err(why) => {
+            warn!(
```
Comment (duncanista) [inline bottlecap/src/tags/lambda/tags.rs:153]: Maybe a debug? I'm afraid this might spam users

File: bottlecap/src/tags/lambda/tags.rs [hunk 4]
```diff
@@ -167,12 +215,15 @@ impl Lambda {
 mod tests {
     use super::*;
     use crate::config::Config;
+    use serial_test::serial;
+    use std::io::Write;
 
     #[test]
+    // #[serial]
```
Comment (duncanista) [inline bottlecap/src/tags/lambda/tags.rs:222]: ?

**Review by duncanista — APPROVED** [review]
LGTM – left some comments!

**Comment by duncanista** [comment]
Tests are failing because a new tag adds size to the metric!

File: bottlecap/src/tags/lambda/tags.rs (hunk 4 above)
Comment (alexgallotta) [reply bottlecap/src/tags/lambda/tags.rs:222]: ops missed this, thanks

File: bottlecap/src/tags/lambda/tags.rs (hunk 2 above)
Comment (alexgallotta) [reply bottlecap/src/tags/lambda/tags.rs:2]: ty! My IDE doesn't do the right thing whenever I add new loggers

**Pull request merged**